vendor/parser-standard/parser.mly

```diff
@@ -1076,7 +1073,7 @@ The precedences must be listed from low to high.
 %nonassoc LBRACKETAT
 %right    COLONCOLON                    /* expr (e :: e :: e) */
 %left     INFIXOP2 PLUS PLUSDOT MINUS MINUSDOT PLUSEQ /* expr (e OP e OP e) */
-%left     PERCENT INFIXOP3 MOD STAR                 /* expr (e OP e OP e) */
+%left     PERCENT SLASH INFIXOP3 MOD STAR                 /* expr (e OP e OP e) */
```

Comment:
I think I understand better what happened here now:

The script works by first creating a patchfile with the diffs between the current `parser-standard` and the copy of these files that were at the tip of `flambda-backend` last time we imported - this contains the intentional ways in which `parser-standard` differs from the compiler's parser (for example, to support parsing typed holes).  Then it replaces `parser-standard`'s files with the copies from the new tip of `flambda-backend` and applies the patchfile to restore the ocamlformat specific changes.

What happened here is it couldn't fully apply the patchfiles because some part of the diff context has changed in flambda-backend (in this particular case, the line `%nonassoc prec_unboxed_product_kind` was added in flambda-backend's parser two lines down).  So it just didn't apply that part of the patch, and created a `.rej` file with the rejected portions.

That all seems pretty sensible, and I don't think there are any updates to the script I would make - it already says loudly that some of the diff couldn't be applied. I am also adding a note in the readme about this.